### ISSUE TITLE
Need help commands chat

### ISSUE THREAD

**Issue by Nizev** [andrewda/node-steam-guide#91]
Hello, how can I make my bot receive purchase and sale orders and the trade is automatic, through commands. example
›buy ›sell

My bot accepts trades but I try to make it respond to commands
sorry for my english, i used translate

**Comment by DentFuse** [comment]
Refer to [Chapter 3.2](https://github.com/Steam-Bot-Basics/node-steam-guide/blob/master/Chapter%203%20-%20User%20Interaction/Chapter%203.2%20-%20Chatting%20With%20Friends/README.md) in Steam-Bot-Basics/node-steam-guide, it's a fork of this repo and has been updated recently.

**Comment by xmwx38** [comment]
I agree with the answer above by @DentFuse.  I am one of a few contributors of [Steam-Bot-Basics/node-steam-guide](https://GitHub.com/Steam-Bot-Basics/node-steam-guide).  Check out [Chapter 3.2](https://github.com/Steam-Bot-Basics/node-steam-guide/blob/master/Chapter%203%20-%20User%20Interaction/Chapter%203.2%20-%20Chatting%20With%20Friends/README.md).
